src/integrations.py

```diff
@@ -335,23 +338,31 @@ class TLSCertificates:
     ca_bundle: str
 
     @classmethod
-    def load(cls, requirer: CertificateTransferRequires) -> "TLSCertificates":
+    def load(
+        cls, requirer: CertificateTransferRequires | V1CertificateTransferRequires
+    ) -> "TLSCertificates":
         """Fetch the CA certificates from all "receive-ca-cert" integrations.
 
         Compose the trusted CA certificates in /etc/ssl/certs/ca-certificates.crt.
         """
+        if isinstance(requirer, V1CertificateTransferRequires):
+            return cls(ca_bundle="\n".join(requirer.get_all_certificates()))
```

Comment:
The requirer should always be `V1CertificateTransferRequires`, we don't need the v0 object as we only used it to access the relation databag.